### DISCUSSION TITLE
Which is better...tabulasharp or camelotsharp?

### DISCUSSION THREAD

**Discussion by jzabroski** [BobLd/tabula-sharp#27] [Q&A]
Hi @BobLd ,

I see you have ported two libraries to C#... tabula and camelot.  Which is better? Do you simply use both and pick the best results?

**Comment by BobLd** [comment]
Hi @jzabroski, thanks for your message.

In its current state, camelot-sharp is only in a pre-release state and I did not finish the port fully... so tabula-sharp is the one to use currently.

I need to find time to go back to camelot-sharp and finish the port, feel free to contribute if you want.
